designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/SearchBar.kt

```diff
@@ -68,10 +68,11 @@ fun SearchBar(
     onQueryChange: (String) -> Unit = {},
     state: InputShellState = InputShellState.FOCUSED,
     modifier: Modifier = Modifier,
+    leadingIcon: @Composable (() -> Unit)? = null,
```

Comment:
Could you add the documentation of the two new parameters?

Reply:
Done!